include/glow/Graph/Graph.h

```diff
@@ -251,6 +251,12 @@ class Function final : public Named {
   Node *createChannelShuffle(llvm::StringRef name, NodeValue input,
                              size_t group, size_t kernel);
 
+  /// Removes single-dimensional entries from the shape of a tensor. The
+  /// parameter axes is a list of positive integeres, indicating the dimensions
+  /// to squeeze
+  Node *createSqueeze(llvm::StringRef name, NodeValue input,
+                      UnsignedArrayRef axes);
```

Comment:
why arrayref?

Reply:
It looks to me that the list of dims passing to the create** functions using this llvm:ArrayRef<size_t> type. Just following the style :P .  In addition,  the ArrayRef is claimed to be safer.

Reply:
I was confused because in the header it's UnsignedArrayRef, but in cpp it's llvm::ArrayRef<size_t>.
It appears that it's us who defines it `using UnsignedArrayRef = llvm::ArrayRef<size_t>;`

Can you do a consistent naming?

Reply:
I see :D  Will modify it.